docs/src/pages/guides/FHIRServerUsersGuide.md

```diff
@@ -1327,23 +1327,65 @@ To enable export to parquet, an administrator must:
+
+| CADF Audit Log Entry Field                           | Description |
+|------------------------------------------------------|-------------|
+|`action`                                              |Action that created the audit event. Possible values are "read", "update", "create", "delete", and "unknown".|
+|`eventTime`                                           |Audit event creation timestamp.|
+|`eventType`                                           |Audit event type. Value is always "activity".|
+|`id`                                                  |Globally unique identifier for the audit event.|
+|`outcome`                                             |Action outcome. Possible values are "success", "failure", "unknown", and "pending".|
+|`typeURI`                                             |TypeURI property of the CADF event entity. Value is always "http://schemas.dmtf.org/cloud/audit/1.0/event".|
+|`attachments`                                         |Note: Contains FHIR server-specific audit event data.|
+|`attachments:contentType`                             |FHIR server-specific audit event data content type. Value is always "application/json".|
+|`attachments:content`                                 |Note: Contents of this field (with its subfields) is encoded as Base64.
+|`attachments:content:request_unique_id`               |Globally unique identifier for the FHIR server request.|
+|`attachments:content:action`                          |FHIR action type. Possible values are "C" (create), "U" (update), "R" (read), "D" (delete), "P" (patch), and "O" (custom operation).|
+|`attachments:content:operation_name`                  |FHIR custom operation name.|
+|`attachments:content:start_time`                      |FHIR request start time.|
+|`attachments:content:end_time`                        |FHIR request end time.|
+|`attachments:content:api_parameters:request`          |FHIR request URL.|
+|`attachments:content:api_parameters:request_status`   |FHIR request HTTP status (e.g. 200).|
+|`attachments:content:data:resource_type`              |Resource type of FHIR resource that was created, updated, or deleted.|
+|`attachments:content:data:id`                         |Resource ID of FHIR resource that was created, updated, or deleted.|
+|`attachments:content:data:version`                    |Updated version of FHIR resource that was created, updated, or deleted.|
+|`attachments:content:batch:resources_read`            |FHIR resource count retrieved on a search request.|
+|`attachments:content:event_type`                      |FHIR event type. Possible values are "fhir-create", "fhir-update", "fhir-patch", "fhir-delete", "fhir-read", "fhir-version-read", "fhir-history", "fhir-search", "fhir-bundle", "fhir-validate", "fhir-metadata", "fhir-configdata", and "fhir-operation".|
+|`attachments:content:description`                     |FHIR event description.|
```

Comment:
what is this?

Reply:
Which field are you asking about specifically?

Reply:
I updated the description of attachments/content/description, in case that was it.

Reply:
yes, thats the one i was going for, sorry to make you guess.  looks good.